action_tutorials/action_tutorials_actions/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 cmake_minimum_required(VERSION 3.5)
-project(action_tutorials)
+project(action_tutorials_actions)
```

Comment:
By convention, we should name this package `action_tutorials_interfaces` I think.